src/components/Cluster/ClusterDetail/NodePool.tsx

```diff
@@ -52,41 +53,108 @@ const NameWrapperDiv = styled.div`
   }
 `;
 
-class NodePool extends Component {
-  state = {
+interface INPViewAndEditName {
+  activateEditMode: () => boolean;
+  name: string;
+}
+
+type TNPViewAndEditName = React.Component<INPViewAndEditName>;
```

Comment:
Types don't have to be prefixed with the `T` letter, only interfaces need to have the `I` prefix

Reply:
We already have a `NPViewAndEditName` for the styled component... Any other onvention we can use here?

Reply:
Maybe `NPViewAndEditNameComponent`?

Reply:
I meant for types, sorry. Any arguments against using T prefix for types? I assume it's not common practice, but ¿Why we shouldn't use T?

Reply:
Well since any other class or function represent a type, with that logic, we'd need to have the `T` prefix for classes and functions, as well... which is kind of annoying.

With other words: we'd use the `User` name when creating a `User` class (which also has the `User` type), it would be kind of weird and unnecessary to use `TUser`. Same with types

Reply:
Makes sense. TY!